controllers/v1/user-extension.js

```diff
@@ -264,4 +264,69 @@ module.exports = class UserExtension extends Abstract {
     })
   }
 
+
+
+   /**
+  * @api {get} /kendra/api/v1/user-extension/read
+  * @apiVersion 1.0.0
+  * @apiName user Extension List
+  * @apiGroup User Extension
+  * @apiHeader {String} X-authenticated-user-token Authenticity token
+  * @apiSampleRequest /kendra/api/v1/user-extension/read
+  * @apiUse successBody
+  * @apiUse errorBody
+  * @apiParamExample {json} Response:
+  * {
+   "status":200,
+   "message":"User extension fetched successfully",
+   "success":true,
+   "result":{
+      "_id":"60a777528b5424712b5faec3",
+      "platformRoles":[
+         {
+            "programs":[
+               "5f34ec17585244939f89f90c",
+               "5f34e44681871d939950bca6"
+            ]
+         },
+         {
+            "programs":[
+               "5f34ec17585244939f89f90c",
+               "5f75b90454670074deacf087"
+            ]
+         },
+         {
+            "programs":[
+               "5f34ec17585244939f89f90c",
+               "5f75b90454670074deacf087"
+            ]
+         }
+      ]
+   }
+}
+**/
+  read(req){
+    return new Promise(async (resolve, reject) => {
+      let userInformation = await userExtensionsHelperV2.userExtensionDocument( {
```

Comment:
@ankitpws What is the diff between https://github.com/shikshalokam/ml-core-service/blob/master/module/user-extension/helperv2.js and https://github.com/shikshalokam/ml-core-service/blob/master/module/user-extension/helper.js Can we not use the same? Also it function can be similar to static programDocuments() in https://github.com/shikshalokam/ml-core-service/blob/master/module/programs/helper.js

Reply:
Removed HelperV2